config.toml

```diff
@@ -65,3 +65,7 @@ disableKinds = ['taxonomy', 'taxonomyTerm']
 
   # Enable search
   BookSearch = true
+
+  # Set logo to Reginald
+  # BookLogo = 'Reginald_hex.png'
```

Comment:
Is left over from trying another method to include the logo?

Reply:
Yes - I went down a bit of a rabbit hole before realising the limits of the hugo-book partials. This and the `<p>` below are remnants of that attempt.